### ISSUE TITLE
Need source details when hystrix fallback is called

### ISSUE THREAD

**Issue by bsushant-athena** [spring-cloud/spring-cloud-netflix#2655]
In ApiGateway we have specified routes as "*" i.e. fallback for all routes. So now I want to log the information about a service or source when any fallback occurs( i.e. which service is down or not responding )so that I'll have exact info and it will be helpful for debugging.

As per discussion with @ryan J Baxter it looks like the enhancement is required. Can somebody provide more details on this or if you have a workaround for this ?

```
GenericZuulRoutesFallbackProvider defaultZuulFallback = new GenericZuulRoutesFallbackProvider();
defaultZuulFallback.setRoute("*");
```

Thanks,
Sushant

**Comment by bsushant-athena** [comment]
hey ,can you let me know how to use this functionality ? and is it safe to use in production ?

**Comment by ryanjbaxter** [comment]
It is in the 2.0.0.BUILD-SNAPSHOTS for now.  It will be included in the next 2.0.0 milestone.  Milestones are not recommended for production use.

**Comment by bsushant-athena** [comment]
@ryanjbaxter  with the latest spring cloud version we are able to get the hystrix fallback logs that prints the service IP which is returning 503.
But we are noticing that if the fallback continues then the stacktrace does not get printed for every 503 so we lose the ability to get the service ip. 
Do you know any other way to either get the 503 stacktrace for each fallback error or somehow print the IP for each fallback error?

We also noticed that `loadbalancer.LoadBalancerContext` class does print the IP  for every case but this class is not getting called in our aws deployment environment where we are using the EIP approach.
Please suggest
